code/modules/shuttle/emergency.dm

```diff
@@ -10,6 +11,7 @@
 	icon_keyboard = "tech_key"
 	var/auth_need = 3
 	var/list/authorized = list()
+	var/last_early_auth = 0
```

Comment:
Make a file called ``yogstation/code/modules/shuttle/emergency.dm`` and put this in there. Also include it in ``yogstation.dme``.